Expanded breadcrumb.

kantrowitz opened the pull request:
Addresses #1694 Expanded breadcrumb.

Commit:
Expanded breadcrumb.

awong-dev commented:
@webinista I pulled down this change. Something seems borked with the plugin code. It seems like maybe an off-by-one error?

webinista commented:
@awong-dev No. The plugin weird and not entirely intuitive, but it works.  

Here's what's going on: the third/last breadcrumb will _always_ match the `title` attribute that's in the corresponding Markdown file. You actually _can't_ override it, despite lines in `_config.yml` that suggest that you can. (Which is why I gave @kantrowitz bad info. Sorry about that.) 

As it turns out, we actually just need to change the title of `_employment/job-seekers/veci.md` from "VECI" to "Veterans Economic Community Initiative." But doing so will cause the breadcrumbs to wrap. I suspect that's why we changed the title to VECI in the first place.

![screen shot 2016-04-06 at 2 46 54 pm](https://cloud.githubusercontent.com/assets/354277/14334017/6ff62634-fc06-11e5-9bb7-2a1b9db62a46.png)

webinista committed:
Updating breadcrumbs section of README. Cleaning up _config.yml

- Clarifying how to update the text of breadcrumbs
- Removing misleading code from _config.yml

emilyville commented:
Breadcrumb wrapping is ok - this already happens a bunch. It's not ideal but this won't be the first!

webinista committed:
Expanding VECI title to Veterans Economic Community Initiative

webinista commented:
Ok. In that case, this PR is ready for a review.

kantrowitz commented:
Should we change this in the readme, then?

emilyville commented on _config.yml:
```diff
@@ -84,16 +84,13 @@ collections:
-    values:
-      breadcrumb_3: "Exposure to Hazardous Materials"
-  - scope:
       path: "disability-benefits/apply-for-benefits/"
```
Why did we remove this here? Not sure if this was intentional - or just an instance of V2 being out of sync with master

emilyville commented on _config.yml:
```diff
@@ -108,10 +105,6 @@ defaults:
-      path: "education/gi-bill/survivors-dependent-assistance"
-    values:
-      breadcrumb_3: "Survivors' and Dependents' Assistance"
-  - scope:
```
Same here

emilyville commented on _config.yml:
```diff
@@ -128,10 +121,6 @@ defaults:
-      path: "education/work-learn/non-traditional"
-    values:
-      breadcrumb_3: "Non-Traditional Options"
-  - scope:
```
and here!

emilyville commented:
I have a hunch the only things meant to change in this PR are the readme and the _employment/job-seekers/veci.md title?

If so, there's an extra bunch of changes in the config file that seem like they snuck in. Back at you guys, @kantrowitz and @webinista

webinista commented:
No. That's correct. I removed those lines from `_config.yml` because they're misleading. The Markdown file titles are what's defining the text in those breadcrumbs, not the `breadcrumb_3` value.

emilyville commented:
great! I should have read the readme more thoroughly. :D Thanks @webinista and I will go ahead and merge - LGTM.
